### DISCUSSION TITLE
status code 500 su getNewIssuedDocumentTotals

### DISCUSSION THREAD

**Discussion by zpj61** [fattureincloud/api#264] [Q&A] [answered]
Ho uno status code 500 su getNewIssuedDocumentTotals e come content  "An unexpected error occurred. Contact support for more information." su un documento tipo invoice. Come possiamo gestire questo? Il json pare corretto a prima vista.

**Comment by valmoz** [comment, marked as answer]
Buongiorno,
le consiglio di contattare il supporto clienti. Con le informazioni che ci ha passato non possiamo fornirle un supporto adeguato.

A disposizione
Mauro
